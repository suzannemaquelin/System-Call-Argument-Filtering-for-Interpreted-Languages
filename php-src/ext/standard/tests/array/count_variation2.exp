*** Testing count() : usage variations ***

-- Iteration 1 --
int(3)

-- Iteration 2 --
int(5)

-- Iteration 3 --
int(3)

-- Iteration 4 --
int(3)

-- Iteration 5 --
int(3)

-- Iteration 6 --
int(3)

-- Iteration 7 --
int(3)

-- Iteration 8 --
int(3)

-- Iteration 9 --
int(3)

-- Iteration 10 --
int(3)

-- Iteration 11 --
int(3)

-- Iteration 12 --
int(5)

-- Iteration 13 --
int(3)

-- Iteration 14 --
int(5)

-- Iteration 15 --
int(3)

-- Iteration 16 --

Warning: count() expects parameter 2 to be integer, string given in %s on line %d
NULL

-- Iteration 17 --

Warning: count() expects parameter 2 to be integer, string given in %s on line %d
NULL

-- Iteration 18 --

Warning: count() expects parameter 2 to be integer, string given in %s on line %d
NULL

-- Iteration 19 --

Warning: count() expects parameter 2 to be integer, string given in %s on line %d
NULL

-- Iteration 20 --

Warning: count() expects parameter 2 to be integer, string given in %s on line %d
NULL

-- Iteration 21 --

Warning: count() expects parameter 2 to be integer, object given in %s on line %d
NULL

-- Iteration 22 --
int(3)

-- Iteration 23 --
int(3)

-- Iteration 24 --

Warning: count() expects parameter 2 to be integer, resource given in %s on line %d
NULL
Done