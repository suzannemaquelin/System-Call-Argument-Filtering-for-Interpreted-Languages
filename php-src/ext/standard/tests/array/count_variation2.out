Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
*** Testing count() : usage variations ***

-- Iteration 1 --
int(3)

-- Iteration 2 --
int(5)

-- Iteration 3 --
int(3)

-- Iteration 4 --
int(3)

-- Iteration 5 --
int(3)

-- Iteration 6 --
int(3)

-- Iteration 7 --
int(3)

-- Iteration 8 --
int(3)

-- Iteration 9 --
int(3)

-- Iteration 10 --
int(3)

-- Iteration 11 --
int(3)

-- Iteration 12 --
int(5)

-- Iteration 13 --
int(3)

-- Iteration 14 --
int(5)

-- Iteration 15 --
int(3)

-- Iteration 16 --

Warning: count() expects parameter 2 to be integer, string given in /root/php-src/ext/standard/tests/array/count_variation2.php on line 88
NULL

-- Iteration 17 --

Warning: count() expects parameter 2 to be integer, string given in /root/php-src/ext/standard/tests/array/count_variation2.php on line 88
NULL

-- Iteration 18 --

Warning: count() expects parameter 2 to be integer, string given in /root/php-src/ext/standard/tests/array/count_variation2.php on line 88
NULL

-- Iteration 19 --

Warning: count() expects parameter 2 to be integer, string given in /root/php-src/ext/standard/tests/array/count_variation2.php on line 88
NULL

-- Iteration 20 --

Warning: count() expects parameter 2 to be integer, string given in /root/php-src/ext/standard/tests/array/count_variation2.php on line 88
NULL

-- Iteration 21 --

Warning: count() expects parameter 2 to be integer, object given in /root/php-src/ext/standard/tests/array/count_variation2.php on line 88
NULL

-- Iteration 22 --
int(3)

-- Iteration 23 --
int(3)

-- Iteration 24 --

Warning: count() expects parameter 2 to be integer, resource given in /root/php-src/ext/standard/tests/array/count_variation2.php on line 88
NULL
Done